src/position.rs

```diff
@@ -30,6 +31,49 @@ pub enum AprsCst {
     Uncompressed,
 }
 
+#[derive(Debug, Copy, Clone, PartialOrd, PartialEq, Ord, Eq)]
+pub enum Precision {
+    TenDegree,
+    OneDegree,
+    TenMinute,
+    OneMinute,
+    TenthMinute,
+    HundredthMinute,
+}
+
+impl Precision {
+    pub(crate) fn num_digits(&self) -> u8 {
```

Comment:
These functions seem pretty useful to downstream users, might want them to be `pub` instead of `pub(crate)`.

Reply:
My only concern is that the # of digits seems fairly specific to APRS. The first two digits are degrees and thus have a different significance than the other digits.

If we went the `AmbiguousValue` route (which I am heavily in favour of) I would be tempted to make `Precision` an internal-only type